src/Components/Navigation/NavBar.jsx

```diff
@@ -10,14 +10,14 @@ function NavBar() {
                 <div className={style.nav_links}>
                     <ul className={style.menu}>
                         <li>
-                            <Link to="/">Home</Link>
+                            <NavLink to="/">Home</NavLink>
                         </li>
                         <li>
-                            <Link to="/api/getAllArticles">Articles</Link>
+                            <NavLink to="/api/getAllArticles">Articles</NavLink>
                         </li>
 
                         <li>
-                            <Link to="/api/getAllTopics">Topics</Link>
+                            <NavLink to="/api/getAllTopics">Topics</NavLink>
```

Comment:
these front-end routes do not need `/api` in them 

i mentioned this [here](https://github.com/AdilBulsari/adil-fe-ncnews/pull/4#issuecomment-1198042988) before - make sure you're addressing all our feedback before merging your code!